api/test/test_events.py

```diff
@@ -144,10 +147,12 @@ def setUp(self):
                 'url': 'https://user.github.io/repository/',
                 'contact': 'sb@sth.edu',
                 'eventbrite_id': None,
+                'tags': [],
             },
         ]
 
-    @patch.object(PublishedEvents, 'request', query_params={}, create=True)
+    @patch.object(PublishedEvents, 'request', query_params=QueryDict(),
```

Comment:
Because the view started using `query_params.getlist` which is specific to `QueryDict`, not "normal" `dict`. This change is compatible with `dict` and prevents AttributeError.